web/cobrands/fixmystreet-uk-councils/assets.js

```diff
@@ -37,6 +37,16 @@ fixmystreet.assets.brent.construct_asset_name = function(id) {
     }
 };
 
+
+fixmystreet.assets.brent.found = function(layer) {
```

Comment:
I've switched to this locally and I'm getting an error message about 'criterion' not being a function - I've put in a fix that tests criterion is a function, but it's almost certainly not the right approach, but need your advice - possible I've got something else wrong.

Reply:
Oh dear, no, perhaps I was wrong in saying this could be done like this! I thought we had something existing that did it, but I cannot find it now, apologies. So road_found is calling not_found with `(this, this.selected_feature)`, but the not found function expects a criterion as 2nd argument, as you have found. We could change in time this to take named arguments instead of positioned ones, but not for now - perhaps your original way was the easiest at present.